File: Find the K-th Character in String Game I/Find the K-th Character in String Game I.cpp
// Solution for Find the K-th Character in String Game I in CPP

class Solution {
public:
    char kthCharacter(int k) {
        string word = "a";
        while (word.length() < k) {
            string temp;
            for(char ch: word) {
                temp += (ch == 'z') ? 'a' : ch + 1;
            }
            word += temp;
        }
        return word[k - 1];
    }
};

File: Find the K-th Character in String Game I/Find the K-th Character in String Game I.py
# Solution for Find the K-th Character in String Game I in PY

class Solution:
    def kthCharacter(self, k: int) -> str:
        word = "a"
        while len(word) < k:
            temp = ""
            for ch in word:
                temp += 'a' if ch == 'z' else chr(ord(ch) + 1)
            word += temp
        return word[k - 1]
